include/receiver_cadmium.hpp

```diff
@@ -79,8 +79,8 @@ class Receiver {
 
         // confluence transition
         void confluence_transition(
-		                TIME e,
-		                typename make_message_bags<input_ports>::type mbs) {
+	    TIME e,
+                typename make_message_bags<input_ports>::type mbs) {
```

Comment:
Before the Time e, there are 4 spaces because there is an open bracket at the end of the previous line, but why is the 4 spaces before typename which is under TIME e?

Reply:
Resolved